### PR TITLE
Track "taps" on the hamburger

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

**Review by gnardini — APPROVED** [review]
🚀

**Review by cajubelt — APPROVED** [review]

File: packages/mobile/src/navigator/DrawerTopBar.tsx [hunk 1]
```diff
@@ -29,14 +31,15 @@ function DrawerTopBar({ middleElement, rightElement, scrollPosition, testID }: P
...
+
   return (
     <Animated.View testID={testID} style={viewStyle}>
-      <TouchableOpacity
-        style={styles.hamburger}
-        // @ts-ignore Only used in a drawer
-        onPress={navigation.toggleDrawer}
-        hitSlop={iconHitslop}
-      >
+      <TouchableOpacity style={styles.hamburger} onPress={hamburgerOnPress} hitSlop={iconHitslop}>
```
Comment (silasbw) [inline packages/mobile/src/navigator/DrawerTopBar.tsx:42]: changing `onPress` from `navigation.toggleDrawer` to `hamburgerOnPress` (or even `() => { navigation.toggleDrawer() }`) causes the snapshot tests to fail with this diff:

```
          <View
            accessible={true}
    -       focusable={false}
    +       focusable={true}
            hitSlop={
              Object {
                "bottom": 10,
                "left": 10,
                "right": 10,
```

That's unexpected and can't figure out why it's happening.  @jeanregisser or @kathaypacific any ideas?

File: packages/mobile/src/navigator/DrawerTopBar.tsx [hunk 2]
```diff
@@ -29,14 +31,15 @@ function DrawerTopBar({ middleElement, rightElement, scrollPosition, testID }: P
     [scrollPosition]
   )
 
+  const hamburgerOnPress = () => {
```
Comment (jeanregisser) [inline packages/mobile/src/navigator/DrawerTopBar.tsx:34]: nit: we use this convention for naming event handlers.
```suggestion
  const onPressHamburger = () => {
```

**Review by jeanregisser — APPROVED** [review]
Thanks!

**Comment by ValoraQA** [comment]
Hey @MuckT, Please let us know do we need to perform testing from our side.

**Pull request merged**